Gemfile.lock

```diff
@@ -1,11 +1,15 @@
 GIT
   remote: https://github.com/tulibraries/guiding_light.git
-  revision: 43d7fab5be0703e8b88de1fd0f8018263bd117ff
+  revision: 10135c504c347e6fe7916f65e00872fd6787bb29
```

Comment:
Nothing will break. Once Guiding Light changes are merged, we'll get the added whenever capability as well as the updated Guiding Light code.